units/stats_amboy.go

```diff
@@ -92,16 +91,6 @@ func (j *amboyStatsCollector) Run(ctx context.Context) {
 			"message": "amboy remote queue stats",
 			"stats":   remoteQueue.Stats(ctx),
 		})
-
-		serviceFlags, err := evergreen.GetServiceFlags()
```

Comment:
It doesn't look like this job really does anything anymore; we'll just be logging work that isn't actually being done. We should probably just remove the whole job.

Reply:
Makes sense, removing